R/config.R

```diff
@@ -67,6 +67,9 @@ vt_use_validation_config <- function(pkg = ".",
     )
   }
 
+  ## add ".here" ref if not a package
+  set_dir_ref(pkg = pkg)
```

Comment:
if this is a package, we should add .validation to .Rbuildignore

Reply:
done. resolved in da12206